src/Monolog/Handler.php

```diff
@@ -73,6 +73,11 @@ protected function doWrite($record): void
                 $scope->setExtra('monolog.context', $monologContextData);
             }
 
+            $monologExtraData = $this->getMonologExtraData($record['extra']);
+            if ([] !== $monologExtraData) {
```

Comment:
Any reason to not use `!empty()` as we did a few lines above? I would prefer it for consistency purposes, if not else

Reply:
Empty is overall considered a dangerous construct, `!==[]` comparison is more strict and shows intent.

I'd rather update the empty few lines above to `[] !==` if you want consistency. WDYT?